strapi/strapi
# fix: language-change-issue

### What does it do?
Fix #18156

Describe the technical changes you did.
Removed the `usePersistentState` and made `lang` local to AssetPreview component.

### Why is it needed?

The media editor dialog was changing the `strapi-admin-language` to `en` which was overriding the locale chosen by the user.

### How to test it?

1. Change the language to anything other than english in `Profile` page and save
2. Then navigate to 'Media Library', select a preuploaded media and click `Edit asset`. Hit refresh, it should not change the language back to english


Thanks,

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
